docs/raw/project.md

```diff
@@ -18,7 +18,8 @@ environment
 
 - Type: `string` 
```

Comment:
Please exclude the whole file from the PR, I have a much more descriptive information waiting on a different PR.